docs/install.md

```diff
@@ -1,10 +1,10 @@
-#Installation and Settings
-##Prerequisites
-The MongoDB module and sub-modules need some configuration to be useful. This
-guide assumes that :
+# Installation and Settings
+## Prerequisites
+The MongoDB module and sub-modules need some configuration to be useful.
+This guide assumes that :
 
-* a [MongoDB][download] 3.0 to 4.0 server instance is already installed, configured, and
-  available to connect to from the Drupal instance.
+* a [MongoDB][download] 3.0 to 4.0 server instance is already installed,
+  configured and available to connect to from the Drupal instance.
 * the site will be running [Drupal][drupal] 8.[56].x, with [Drush][drush] 8.x.
 * the [mongodb][mongodb] (not [mongo][mongo]) PHP extension version 1.1.7 or
   later is installed and configured.
```

Comment:
Since 8.x-2.x is the main branch for this suite, it is better to merge mkdocs to 8.x-2.x once we are done with this ticket and current updates. 
   However in any future documentation related issues, we can mention  that the documentation updates to be done on mkdocs branch rather than in the main branch so that at any point it will be easy for getting the commits together on how the documentation has merged.